configs/phalconphp.json

```diff
@@ -1,7 +1,58 @@
 {
   "index_name": "phalconphp",
   "start_urls": [
-    "https://docs.phalconphp.com/4.0/"
+    {
+      "url": "https://docs.phalconphp.com/(?P<version>.*?)/(?P<lang>.*?)/",
+      "variables": {
+        "lang": [
+          "en",
+          "cs-cz",
+          "de-de",
+          "el-gr",
+          "es-es",
+          "fa-ir",
+          "fr-fr",
+          "id-id",
+          "it-it",
+          "ja-jp",
+          "nl-nl",
+          "pl-pl",
+          "pt-br",
+          "ru-ru",
+          "sr-sp",
+          "th-th",
+          "tr-tr",
+          "uk-ua",
+          "zh-cn,"
+        ],
+        "version": [
+          "latest",
+          "4.0"
+        ]
+      }
+    },
+    {
+      "url": "https://docs.zephir-lang.com/3.4/(?P<lang>.*?)/",
+      "variables": {
+        "lang": [
+          "en",
+          "cs-cz",
+          "de-de",
+          "es-es",
+          "fa-ir",
+          "fr-fr",
+          "id-id",
+          "it-it",
+          "ja-jp",
+          "pl-pl",
+          "ru-ru",
+          "tr-tr",
+          "uk-ua",
+          "ur-in",
+          "zh-cn"
+        ]
```

Comment:
Please add an extra_attributes for the version here just like https://github.com/algolia/docsearch-configs/blob/f3b40e5435fa5f289cfbd758eb5f4e4286b3f6a6/configs/brew_all.json#L10

Reply:
Added the extra attributes